### ISSUE TITLE
Why Mapbox over free and open source mapping libraries?

### ISSUE THREAD

**Issue by abarciauskas-bgse** [NASA-IMPACT/veda-ui#69]
In response to the question "Why Mapbox?" @danielfdsilva  helped provide the following explanation:

***

**TL;DR:**
* Best match for the requirements (at the time)
* Deep knowledge
* Preexisting code
* Relationship with the company

At the beginning of the project we looked at what was out there to understand if there was an alternative to mapbox (https://github.com/NASA-IMPACT/covid-dashboard/issues/557).

We were looking to have different types of projections while supporting vector layers and data-driven styling.

Because we were looking for vector layer support, mapbox was a strong choice and no other library is as powerful at doing data-driven styling.

Initially it lacked projections support, but then they added them - and globe projection is coming along.

This, tied with our extensive knowledge of mapbox, and close relationship made it an easy sell.

Now, looking at what we are working with right now, it seems that vector layers are not really needed - I’m actually not sure about this, but all our conversations seem to refer to tiled datasets.

This seemingly tiny change, means we could reassess and go with a different library.

***

It sounds like we may consider moving to another mapping client. From what I understand there are some deal-breaker requirements (and probably some that I am missing):

* Open source (if we're deciding not to use Mapbox, seems likely we would want to use an open source solution)
* Support for projections required for the thematic areas of NASA's Earth Observation users (web mercator, polar, possibly others)
* Support for 3D projections

What are some other important features we need to consider in alternatives? @danielfdsilva also mentioned data-driven styling for example.

Is [CesiumJS]()https://cesium.com/platform/cesiumjs) worth exploring as well?

**Comment by nerik** [comment]
So I gave it some thoughts: https://github.com/developmentseed/how/issues/376
